### ISSUE TITLE
GHC-8 version is not on hackage (also applies to hyperloglog)

### ISSUE THREAD

**Issue by alexanderkjeldaas** [ekmett/approximate#6]
The latest bounds fixes are not on hackage, and thus doesn't work with lts-7.0

```
In the dependencies for approximate-0.2.2.3:
    binary-0.8.3.0 must match >=0.5 && <0.8 (latest applicable is 0.7.6.1)
    comonad-5 must match >=3 && <5 (latest applicable is 4.3)
    pointed-5 must match >=3 && <5 (latest applicable is 4.2.0.2)
needed due to subhask-0.1.1.0 -> approximate-0.2.2.3

In the dependencies for hyperloglog-0.4.0.4:
    binary-0.8.3.0 must match >=0.5 && <0.8 (latest applicable is 0.7.6.1)
    comonad-5 must match >=4 && <5 (latest applicable is 4.3)
needed due to subhask-0.1.1.0 -> hyperloglog-0.4.0.4
```

**Comment by thomasdziedzic** [comment]
This really needs a new version because the binary bump is required for ghc 8.0

**Comment by RyanGlScott** [comment]
Fixed in [`approximate-0.3`](http://hackage.haskell.org/package/approximate-0.3).
